pkgs/misc/uboot/default.nix

```diff
@@ -435,6 +412,31 @@ in {
     defconfig = "rpi_4_defconfig";
     extraMeta.platforms = ["aarch64-linux"];
     filesToInstall = ["u-boot.bin"];
+    extraConfig = ''
+      CONFIG_CMD_NVME=y
+      CONFIG_NVME=y
+      CONFIG_NVME_PCI=y
+      CONFIG_USB_STORAGE=y
+      CONFIG_USB_FUNCTION_MASS_STORAGE=y
+      CONFIG_USB_EHCI_HCD=y
+      CONFIG_USB_EHCI_GENERIC=y
+      CONFIG_USB_OHCI_HCD=y
+
+      CONFIG_USB_XHCI_BRCM=y
+    '';
```

Comment:
[implementation detail]

These *normally* would have to be part of the changeset sent to mainline, and applied on the defconfigs; assuming mainline contributions.

Reply:
Removed to stay close to mainline.